### PR TITLE
Feature/display trips

### PR DESCRIPTION
**Pull request by justincanthony**
- [ ] fix 
- [x] feature?

## What does it change?
- Creates functions to display a traveler's future approved trips and destinations
- adjusts methods on traveler class to manage displaying data
- Installs dayjs
- 

## What does it fix?
- 

## Where should reviewer start?
- Traveler.js 
- domUpdates.js
- Destinations.js - NEEDS to be tested for properties!!! 

## How can this be tested?
-  run npm test on Traveler class
- running local server and checking for errors in the console - there is a clickable eventlistener on the menu-bar class inside the header to load the page

### CODE REVIEW

File: src/Destination.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 class Destination {
-  constructor(destinations) {
-    this.destinations = destinations;
+  constructor(destination) {
+    this.id = destination.id;
+    this.destination = destination.destination;
+    this.estimatedLodgingCostPerDay = destination.estimatedLodgingCostPerDay;
```
Comment (MTCawthray) [inline src/Destination.js:5]: Nice to have these properties available!

File: src/Traveler.js [hunk 2]
```diff
@@ -17,15 +17,22 @@ class Traveler {
...
   }
 
+  getDestinationIDs(data) {
+    return this.trips.map((trip) => trip.destinationID);
+  }
+
   getPastTrips(date) {
     return this.trips.filter((trip) => {
-      if (dayjs(date).isAfter(dayjs(trip.date))) {
+      if (dayjs(date).isAfter(dayjs(trip.date)) && trip.status === 'approved') {
```
Comment (MTCawthray) [inline src/Traveler.js:35]: Smart filter methods here.

File: src/domUpdates.js [hunk 3]
```diff
@@ -1,28 +1,83 @@
...
 };
 
-
 const displayTravelerData = () => {
-
-}
+  document.getElementById('tripper-name').innerText = traveler.name;
+};
 
 const displayDestinationData = () => {
```
Comment (MTCawthray) [inline src/domUpdates.js:18]: I like how dynamic this is.

File: test/Traveler-test.js [hunk 4]
```diff
@@ -59,7 +59,14 @@ describe('Traveler', () => {
     ]);
   });
 
-  it.only('should be able to get pending trips', () => {
+  it('should be able to retrieve their destination Ids', () => {
+    agency = new Agency(travelersTestData, tripTestData, destinationTestData);
```
Comment (MTCawthray) [inline test/Traveler-test.js:63]: Could these agency instantiations be done inside of the before each to save you some lines?

**Review by MTCawthray — APPROVED** [review]
Your code looks very clean and easy to follow. I like the way you set up your class structure too. All of the methods seem to make getting the data you need really efficient. I didn't find much I would change at this point, other than perhaps doing an instantiation of your agency class in the beforeEach of your Traveler-test file. That is just a housekeeping thing though and of no major importance! 

Great work so far!

File: test/Traveler-test.js (hunk 4 above)
Comment (justincanthony) [reply test/Traveler-test.js:63]: You are right, at first i didnt need them, but when I changed the class structure, I kept needing them. Thanks for catching that

**Pull request merged**